docs/dotnet/dotnet-signalr-metrics.md

```diff
@@ -8,60 +8,109 @@ This document defines semantic conventions for SignalR metrics emitted by .NET c
 
 <!-- toc -->
 
-- [Metric: `signalr_http_transport.connection.duration`](#metric-signalr_http_transportconnectionduration)
-- [Metric: `signalr_http_transport.active_connections`](#metric-signalr_http_transportactive_connections)
+- [Metric: `signalr.http.server.connection.duration`](#metric-signalrhttpserverconnectionduration)
+- [Metric: `signalr.http.server.active_connections`](#metric-signalrhttpserveractive_connections)
 
 <!-- tocstop -->
 
-## Metric: `signalr_http_transport.connection.duration`
+## Metric: `signalr.http.server.connection.duration`
 
 **Status**: [Experimental][DocumentStatus]
 
 This metric is required.
 
-<!-- semconv metric.signalr.server.connection.duration(metric_table) -->
+<!-- semconv metric.signalr.http.server.connection.duration(metric_table) -->
 | Name     | Instrument Type | Unit (UCUM) | Description    |
 | -------- | --------------- | ----------- | -------------- |
-| `signalr_http_transport.connection.duration` | Histogram | `s` | The duration of connections on the server. [1] |
+| `signalr.http.server.connection.duration` | Histogram | `s` | The duration of the HTTP connections on the server. [1] |
 
-**[1]:** **TODO: why http_transport? can we report protocol name as dimension and not a metric name?**
-**TODO: we should do signalr.server**
+**[1]:** Only reported for HTTP SignalR transport.
 Corresponding `EventCounter` name is `connections-duration`; Meter name is `Microsoft.AspNetCore.Http.Connections`
 <!-- endsemconv -->
 
-<!-- semconv metric.signalr.server.connection.duration(full) -->
+<!-- semconv metric.signalr.http.server.connection.duration(full) -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
-| `signalr.http_transport.status` | string | TODO [1] | `TODO` | Recommended |
-| `signalr.http_transport.transport` | string | TODO [2] | `TODO` | Recommended |
+| `signalr.signalr.transport` | string | SignalR transport - https://github.com/dotnet/aspnetcore/blob/main/src/SignalR/docs/specs/TransportProtocols.md [1] | `ServerSentEvents` | Conditionally Required: if HTTP SignalR transport is used |
+| `dotnet.error.code` | string | General-purpose error code reported by .NET, as a starter it supports terminal states of .NET task https://learn.microsoft.com/en-us/dotnet/api/system.threading.tasks.taskstatus. | `Canceled`; `RanToCompletion` | Recommended |
+| `http.response.status_code` | int | [HTTP response status code](https://tools.ietf.org/html/rfc7231#section-6). | `200` | Conditionally Required: If and only if one was received/sent. |
```

Comment:
is there need for specific status code since this is http-specific metric?

Reply:
I don't think there is but its my research queue.

Reply:
I think we can remove this.